mycity/mycity/intents/food_truck_intent.py

```diff
@@ -0,0 +1,190 @@
+                response = f"I found {count} food trucks within a mile " \
+                    "from your address! "
+                for i in range(count):
+                    response += f"{nearby_food_trucks[0][1]} is located at " \
+                        f"{nearby_food_trucks[i][2]} and " \
+                        f"{nearby_food_trucks[i][3]}, from " \
+                        f"{nearby_food_trucks[i][4]} to " \
+                        f"{nearby_food_trucks[i][5]}, "
+                mycity_response.output_speech = response
+
+            elif count > 3:
+                response = f"I found {count} food trucks within a mile " \
+                    "from your address! Here are the first three: "
+                for i in range(3):
+                    response += f"{nearby_food_trucks[i][1]} is located at "\
+                        f"{nearby_food_trucks[i][2]} and " \
+                        f"{nearby_food_trucks[i][3]}, from " \
+                        f"{nearby_food_trucks[i][4]} to " \
+                        f"{nearby_food_trucks[i][5]}, "
+                response += "Would you like to hear more?"
+                mycity_response.output_speech = response
+
+        except InvalidAddressError:
+            address_string = address
+            if zip_code:
+                address_string = address_string + " with zip code {}"\
+                    .format(zip_code)
+            mycity_response.output_speech = \
+                speech_constants.ADDRESS_NOT_FOUND.format(address_string)
+            mycity_response.dialog_directive = "ElicitSlotFoodTruck"
+            mycity_response.reprompt_text = None
+            mycity_response.session_attributes = \
+                mycity_request.session_attributes
+            mycity_response.card_title = "Food Trucks"
+            mycity_request = clear_address_from_mycity_object(mycity_request)
+            mycity_response = clear_address_from_mycity_object(mycity_response)
+            return mycity_response
+
+        except BadAPIResponse:
+            mycity_response.output_speech = \
+                "Hmm something went wrong. Maybe try again?"
+
+        except MultipleAddressError:
+            mycity_response.output_speech = \
+                speech_constants.MULTIPLE_ADDRESS_ERROR.format(address)
+            mycity_response.dialog_directive = "ElicitSlotZipCode"
+
+    else:
+        logger.error("Error: Called food_truck_intent with no address")
+        mycity_response.output_speech = "I didn't understand that address, " \
+                                        "please try again"
+
+    # Setting reprompt_text to None signifies that we do not want to reprompt
+    # the user. If the user does not respond or says something that is not
+    # understood, the session will end.
+    mycity_response.reprompt_text = None
+    mycity_response.session_attributes = mycity_request.session_attributes
+    mycity_response.card_title = "Food Trucks"
+
+    return mycity_response
```

Comment:
Nitpick: PEP8 recommends newline character at the end of every Python file. So, newline here and other touched files